week-2/mandatory/3-api/server.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const app = express();
+
+const { Pool } = require("pg");
+
+const db = new Pool({
+  user: "admin", // replace with you username
+  host: "localhost",
+  database: "cyf_ecommerce",
+  password: "password",
+  port: 5432,
+});
+
+app.get("/customers", (req, res) => {
+  db.query("SELECT * FROM customers", (error, result) => {
+    res.json(result.rows);
+  });
+});
+
+app.get("/suppliers", (req, res) => {
+    db.query("SELECT * FROM suppliers", (error, result) => {
+        res.json(result.rows);
+      });
+});
+
+app.get("/products", (req, res) => {
+    db.query("SELECT p.product_name, a.unit_price , s.supplier_name from products p join product_availability a on p.id = a.prod_id join suppliers s on s.id = a.supp_id", (error, result) => {
+        res.json(result.rows);
+      });
+});
+
```

Comment:
Ideally handle the error case - there can sometimes be problems with database connectivity, so you should return a suitable error message in this case. Also, return an appropriate status code for the successful responses and the error responses

Reply:
I made some basic checking and pushed it github if you can review it.

Reply:
Yes, that's exactly right.